src/content/i18n/hi.yml

```
@@ -0,0 +1,124 @@
+a11y.sectionLink: शीर्षक # "Section titled"
+# Site settings
+site.title: Astro दस्तावेज़ीकरण # "Astro Documentation"
+site.og.imageSrc: /default-og-image.png?v=1
+site.og.imageAlt: एक तारों भरे अंतरिक्ष में Astro लोगो, जिसमें दाएँ से एक जामुनी शनि ग्रह की तरह ग्रह तैरता है # "astro logo on a starry expanse of space, with a purple saturn-like planet floating in the right foreground"
+# Left Sidebar
+leftSidebar.sponsoredBy: समर्थित करने वाला # "Sponsored by"
+# Right Sidebar
+rightSidebar.community: समुदाय # "Community"
+rightSidebar.joinDiscord: हमसे Discord में शामिल हों # "Join us on Discord"
+rightSidebar.readBlog: हमारे ब्लॉग पोस्ट पढ़ें # "Read our blog posts"
+rightSidebar.contribute: योगदान करें # "Contribute"
+rightSidebar.translatePage: इस पृष्ठ को अनुवाद करें # "Translate this page"
+# Footer
+footer.privacyPolicy: गोपनीयता नीति # "Privacy Policy"
+# Used in `<Since>`: Added in: v0.24.0 [NEW]
+since.addedIn: 'जोड़ा गया:' # "Added in:"
+since.new: नया # "New"
+since.beta: बीटा # "Beta"
+# Installation Guide
+install.autoTab: स्वचालित CLI # "Automatic CLI"
+install.manualTab: मैन्युअल सेटअप # "Manual Setup"
+# `<DeployGuidesNav>` vocabulary
+deploy.sectionTitle: पृष्ठ संचालन मार्गदर्शिकाएँ # "Deployment Guides"
+deploy.altSectionTitle: अधिक पृष्ठ संचालन मार्गदर्शिकाएँ # "More Deployment Guides"
+deploy.ssrTag: SSR # "SSR"
+deploy.staticTag: Static # "Static"
+# CMS Guides vocabulary
+cms.navTitle: अधिक CMS मार्गदर्शिकाएँ # "More CMS guides"
+# Media Guides vocabulary
+media.navTitle: अधिक DAM मार्गदर्शिकाएँ # "More DAM guides"
+# Migration Guides vocabulary
+migration.navTitle: अधिक माइग्रेशन मार्गदर्शिकाएँ # "More migration guides"
+# Recipes vocabulary
+recipes.navTitle: अधिक रेसिपी # "More recipes"
+# `<RecipeLinks>` vocabulary
+recipesLink.singular: 'संबंधित रेसिपी:' # "Related recipe:"
+recipesLink.plural: संबंधित रेसिपीस # "Related recipes"
+# 404 Page
+404.title: नहीं मिला # "Not Found"
+404.content: यह पेज हमारे सौरमंडल में नहीं है। # "This page isn’t in our solar system."
+404.linkText: मुझे घर ले चलो। # "Take me home."
+# Integrations vocabulary
+integrations.changelog: परिवर्तन लॉग # "Changelog"
+integrations.footerTitle: अधिक एकीकरण # "More Integrations"
+integrations.renderers: UI फ्रेमवर्क # "UI Frameworks"
+integrations.adapters: SSR एडेप्टर # "SSR Adapters"
+integrations.others: अन्य एकीकरण # "Other integrations"
+integrations.more: अधिक एकीकरण, //"More integrations"
```

Oh and spotting this too now:

```suggestion
integrations.more: अधिक एकीकरण # "More integrations"
```